roles/content-creators/README.md

```diff
@@ -0,0 +1,43 @@
+<p align="center"><img src="img/content-creators.svg"></p>
+
+<div align="center">
+  <h4>This is a guide for prospective Content Creators on the latest
+  <a href="https://testnet.joystream.org/pioneer">Joystream Testnet</a>.<h4>
+</div>
+
+
+# Table of Contents
+<!-- TOC START min:1 max:3 link:true asterisk:false update:true -->
+- [Table of Contents](#table-of-contents)
+- [Overview](#overview)
+- [Instructions](#instructions)
+  - [Get Started](#get-started)
+  - [Uploading Content](#uploading-content)
+  - [Content Restrictions](#content-restrictions)
+<!-- TOC END -->
+
+
+# Overview
+
+This page will contain all information on how to act as a `Content Creator`, and how to perform the various tasks required for the job.
+
+# Instructions
+Unlike some of the other current and future roles on the Joystream Platform, becoming a `Content Creator`
+requires no extra software. Everything can be done in the browser, by going [here](http://testnet.joystream.org).
+
+## Get Started
+If you want to upload content as a `Content Creator`, you need to become a `Member`. Instructions for this 
+can be found [here](https://github.com/JoyStream/helpdesk/#get-started).
+
+## Uploading Content
+Once you are a member, it is extremely simple to get started uploading content to the platform.
+
+First, visit the `Media` tab in the sidebar. You can then click on `New channel` in the top navigation bar to make your
```

Comment:
Add that we are currently experiencing some issues with content uploading in chromium based browsers!